src/pages/index.tsx

```diff
@@ -369,197 +283,137 @@ const HomePage = ({
-            </SectionDecription>
-            <ButtonLinkRow>
-              <ButtonLink href="/defi/">
-                <Translation id="page-index:page-index-defi-button" />
-              </ButtonLink>
-            </ButtonLinkRow>
-          </FeatureContent>
-          <ImageContainer>
-            <Image
-              src={impact}
-              alt={t("page-index:page-index-defi-image-alt")}
-              width={700}
-            />
-          </ImageContainer>
-        </Row>
-      </MainSectionContainer>
-      {/* NFT Section */}
-      <MainSectionContainer containerBg="homeBoxMint">
-        <Row isReversed>
-          <FeatureContent>
-            <SectionHeading>
-              <Translation id="page-index:page-index-nft" />
-            </SectionHeading>
-            <SectionDecription>
-              <Translation id="page-index:page-index-nft-description" />
-            </SectionDecription>
-            <ButtonLinkRow>
-              <ButtonLink href="/nft/">
-                <Translation id="page-index:page-index-nft-button" />
-              </ButtonLink>
-            </ButtonLinkRow>
-          </FeatureContent>
-          <ImageContainer>
-            <Image
-              src={infrastructure}
-              alt={t("page-index:page-index-nft-alt")}
-              width={700}
-            />
-          </ImageContainer>
-        </Row>
-      </MainSectionContainer>
-      {/* Internet Section */}
-      <MainSectionContainer containerBg="homeBoxPink">
-        <Box ps={{ lg: 8 }}>
-          <Row>
-            <FeatureContent>
-              <SectionHeading>
-                <Translation id="page-index:page-index-internet" />
-              </SectionHeading>
-              <SectionDecription>
-                <Translation id="page-index:page-index-internet-description" />
-              </SectionDecription>
-              <ButtonLinkRow>
-                <ButtonLink href="/dapps/?category=technology">
-                  <Translation id="page-index:page-index-internet-button" />
-                </ButtonLink>
-                <ButtonLink href="/wallets/" variant="outline" isSecondary>
-                  <Translation id="page-index:page-index-internet-secondary-button" />
+              {description}
+            </SvgButtonLink>
+          ))}
+        </div>
+
+        <BentoBoxes />
+
+        <HomeSection
+          tag="Activity"
+          title="The strongest ecosystem"
+          imgSrc={activityImage}
+        >
+          <div className="mt-16 lg:mt-32">
+            <p className="mt-8 text-xl font-bold">
+              Activity from all Ethereum networks
+            </p>
+            <StatsBoxGrid metricResults={metricResults} />
+          </div>
+        </HomeSection>
+
+        <HomeSection
+          tag="Learn"
+          title="Understanding Ethereum"
+          imgSrc={learnImage}
+          isFlipped
+        >
+          <div className="flex flex-col space-y-16 lg:space-y-32">
```

Comment:
Related to a previous comment, would be nice to align on how we set the spacing for layouts. We are currently using `gap` instead of margins.